Common/Product/SharedProject/Telemetry/TelemetryEvents.cs

```diff
@@ -0,0 +1,24 @@
+﻿//*********************************************************//
+//    Copyright (c) Microsoft. All rights reserved.
+//    
+//    Apache 2.0 License
+//    
+//    You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//    
+//    Unless required by applicable law or agreed to in writing, software 
+//    distributed under the License is distributed on an "AS IS" BASIS, 
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or 
+//    implied. See the License for the specific language governing 
+//    permissions and limitations under the License.
+//
+//*********************************************************//
+
+namespace Microsoft.VisualStudioTools.Telemetry {
+    /// <summary>
+    /// Telemetry event names
+    /// </summary>
+    internal static class TelemetryEvents {
+        public const string ProjectImported = "ProjectImported";
+    }
+}

```

Comment:
I meant to move this to Nodejs/Common, not Common/* alongside the rest of the dlls. 

Again, ```Common/*``` are shared components with Python Tools, and pretty much anyone who's forked us.

Nodejs/Common are shared components within the Nodejs product that we can reference from different NTVS projects.